pkg/registry/generic/etcd/etcd_test.go

```diff
@@ -339,6 +339,53 @@ func TestEtcdUpdate(t *testing.T) {
 
 }
 
+func TestNoOpUpdates(t *testing.T) {
+	server, registry := NewTestGenericEtcdRegistry(t)
```

Comment:
if possible, can we also have a test that inserts an old incorrect record (containing resourceVersion) into etcd, makes sure a no-op update is ignored, and an actual update is persisted?